org.qbittorrent.qBittorrent.yaml

```diff
@@ -53,6 +53,7 @@ modules:
       - -DCMAKE_BUILD_TYPE=RelWithDebInfo
       - -DCMAKE_INTERPROCEDURAL_OPTIMIZATION=ON
       - -Ddeprecated-functions=OFF
+      - -DQT6=ON
```

Comment:
You are passing this to `libtorrent` module while you should to `qbittorent` module below.

Reply:
> You are passing this to `libtorrent` module while you should to `qbittorent` module below.

How can I do that?

Reply:
@iMonZ move that line to config-opts under qbittorrent and it should compile.